### PR TITLE
Enable C++ implementations of deep-learning toolkits by default

### PR DESCRIPTION
**Pull request by nickjong**

### CODE REVIEW

File: src/python/turicreate/toolkits/_internal_utils.py [hunk 1]
```diff
@@ -32,10 +32,15 @@
               UnityGraphProxy: (lambda x: _SGraph(_proxy=x))}
 
 def _read_env_var_cpp(var_name):
+    """
+    Reads environment variables used to enable/disable the new C++
+    implementations of existing Python toolkits.
+    """
+    # Default to using the C++ implementation for toolkits that have one.
+    # TODO: Remove the old implementations, and this function, once the dust has
```
Comment (srikris) [inline src/python/turicreate/toolkits/_internal_utils.py]: Can you file a GitHub issue for this todo :)

Comment (nickjong) [reply src/python/turicreate/toolkits/_internal_utils.py]: Sorry, I don't believe in GH issues that are pure code-cleanup. They don't get done. Seeing a reminder in the code is more useful, IMHO 😛

Comment (srikris) [reply src/python/turicreate/toolkits/_internal_utils.py]: I wasn't suggesting you remove the comment :)

**Comment by nickjong** [comment]
Merging now to unblock testing, since the affected tests are all marked as skipped or xfail anyway

**Pull request merged**